.azure-pipelines-templates/trace_validation.yml

```diff
@@ -6,17 +6,31 @@ steps:
       python3 ./tla/install_deps.py
     displayName: "Install TLA dependencies"
 
-  - script: |
-      set -ex
-      set -o pipefail
-      cd tla/
-      mkdir -p traces/consensus
-      mv ../build/consensus traces/
-      parallel 'JVM_OPTIONS=-Dtlc2.tool.queue.IStateQueue=StateDeque JSON={} ./tlc.sh -dump dot,constrained,colorize,actionlabels {}.dot -dumpTrace tla {}.trace.tla -dumpTrace json {}.trace.json consensus/Traceccfraft.tla' ::: $(ls traces/consensus/*.ndjson)
-    displayName: "Run trace validation"
+  - ${{ if eq(parameters.suffix, 'Tracing') }}:
+      - script: |
+          set -ex
+          set -o pipefail
+          cd tla/
+          mkdir -p traces/consensus
+          mv ../build/consensus traces/
+          parallel 'JVM_OPTIONS=-Dtlc2.tool.queue.IStateQueue=StateDeque JSON={} ./tlc.sh -dump dot,constrained,colorize,actionlabels {}.dot -dumpTrace tla {}.trace.tla -dumpTrace json {}.trace.json consensus/Traceccfraft.tla' ::: $(ls traces/consensus/*.ndjson)
```

Comment:
Out of curiosity, is DFS search (`-Dtlc2.tool.queue.IStateQueue=StateDeque`) required to trace validate this spec?

Reply:
Probably not, this is absolutely copy pasted from the consensus TV.

Reply:
Wait, that _is_ the consensus TV. This is not a change, I've just done some directory renaming. Let's do a pass on the consensus TV separately, and keep this PR focused on consistency.

Reply:
Obviously beyond this PR, but it could be an interesting experiment to let non-determinism handle log ordering and see how quickly DFS finds a trace.